lib/protocol.schema.json

```diff
@@ -700,6 +712,27 @@
                 "status_message",
                 "last_online"
             ]
+        },
+        "PeerInfo": {
+            "type": "object",
+            "properties": {
+                "number": { "type": "number" },
+                "public_key": { "type": "string" },
+                "name": { "type": "string" }
+            },
+            "required": [ "number", "public_key", "name" ]
+        },
+        "ConferenceInfo": {
+            "type": "object",
+            "properties": {
+                "number": { "type": "number" },
+                "kind": { "enum": ["Text", "Av"] },
```

Comment:
$ref?

Reply:
Enums are unwrapped everywhere. Maybe I should move them in separate definitions, but this I'll do it later (in a separate PR)